src/hierarchies/lin_reg_uni_hierarchy.h

```diff
@@ -34,6 +55,12 @@ class LinRegUniHierarchy
     like->set_state(state);
   };
 
+  //! Evaluates the log-marginal distribution of data in a single point
+  //! @param hier_params  Container of (prior or posterior) hyperparameter
```

Comment:
I would specify that this is a pointer to the container. There's two more occurrences of this in this PR, besides here